github/messages.go

```diff
@@ -38,27 +38,29 @@ const (
 var (
 	// eventTypeMapping maps webhooks types to their corresponding go-github struct types.
 	eventTypeMapping = map[string]string{
-		"commit_comment":              "CommitCommentEvent",
-		"create":                      "CreateEvent",
-		"delete":                      "DeleteEvent",
-		"deployment":                  "DeploymentEvent",
-		"deployment_status":           "DeploymentStatusEvent",
-		"fork":                        "ForkEvent",
-		"gollum":                      "GollumEvent",
-		"issue_comment":               "IssueCommentEvent",
-		"issues":                      "IssuesEvent",
-		"member":                      "MemberEvent",
-		"membership":                  "MembershipEvent",
-		"page_build":                  "PageBuildEvent",
-		"public":                      "PublicEvent",
-		"pull_request_review_comment": "PullRequestReviewCommentEvent",
-		"pull_request":                "PullRequestEvent",
-		"push":                        "PushEvent",
-		"repository":                  "RepositoryEvent",
-		"release":                     "ReleaseEvent",
-		"status":                      "StatusEvent",
-		"team_add":                    "TeamAddEvent",
-		"watch":                       "WatchEvent",
+		"commit_comment":                        "CommitCommentEvent",
+		"create":                                "CreateEvent",
+		"delete":                                "DeleteEvent",
+		"deployment":                            "DeploymentEvent",
+		"deployment_status":                     "DeploymentStatusEvent",
+		"fork":                                  "ForkEvent",
+		"gollum":                                "GollumEvent",
+		"integration_installation":              "IntegrationEvent",
+		"integration_installation_repositories": "IntegrationEvent",
```

Comment:
Maybe I'm missing something, but why are these two events/webhook types combined into a single shared struct type? According to https://developer.github.com/early-access/integrations/webhooks/, they have different fields.

Why not have `IntegrationInstallationEvent` and `IntegrationInstallationRepositoriesEvent`?

Somewhat similar to `pull_request_review_comment` and `pull_request` mapping to their own separate types.

Are they different or special in a way? I'm not very familiar with the details here, so I'm asking.